features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/Rum.kt

```diff
@@ -20,6 +20,7 @@ import com.datadog.android.rum.internal.RumAnonymousIdentifierManager
 import com.datadog.android.rum.internal.RumFeature
 import com.datadog.android.rum.internal.metric.SessionEndedMetricDispatcher
 import com.datadog.android.rum.internal.monitor.DatadogRumMonitor
+import com.datadog.android.rum.internal.utils.percent
```

Comment:
unused import

Reply:
we could, but the problem is commit signing - commit will be amended, but bot cannot sign it ootb.